golang/btcspv/types.go

```diff
@@ -28,11 +28,14 @@ type SPVProof struct {
 	Vin               HexBytes      `json:"vin"`
 	Vout              HexBytes      `json:"vout"`
 	Locktime          HexBytes      `json:"locktime"`
-	TxID              Hash256Digest `json:"tx_id"`
+	TxID              []byte        `json:"tx_id"`
```

Comment:
don't change types here, it'll break the serialization stuff

Reply:
Fixed